src/MaterialType.java

```diff
@@ -0,0 +1,109 @@
+public class MaterialType {
+    /**
+     * Unique identifier
+     */
+    private  int id;
+    /**
+     * Material name
+     */
+    private String name;
+    /**
+     * Material description
+     */
+    private String description;
+    /**
+     * Icon URL to be downloaded from S3 amazon server
+     */
+    private String iconURL;

```

Comment:
even if its a minor detail - probably not worth to copy paste a code, write it again, in your own way to just remember better.